app/src/main/java/org/mozilla/fenix/sync/ext/ErrorType.kt

```diff
@@ -1,40 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.fenix.sync.ext
-
-import androidx.annotation.StringRes
-import androidx.navigation.NavController
-import mozilla.components.feature.syncedtabs.view.SyncedTabsView.ErrorType
-import org.mozilla.fenix.R
-import org.mozilla.fenix.sync.SyncedTabsAdapter
-
-/**
- * Converts the error type to the appropriate matching string resource for displaying to the user.
- */
-fun ErrorType.toStringRes() = when (this) {
-    ErrorType.MULTIPLE_DEVICES_UNAVAILABLE -> R.string.synced_tabs_connect_another_device
-    ErrorType.SYNC_ENGINE_UNAVAILABLE -> R.string.synced_tabs_enable_tab_syncing
-    ErrorType.SYNC_UNAVAILABLE -> R.string.synced_tabs_sign_in_message
-    ErrorType.SYNC_NEEDS_REAUTHENTICATION -> R.string.synced_tabs_reauth
-    ErrorType.NO_TABS_AVAILABLE -> R.string.synced_tabs_no_tabs
-}
```

Comment:
I am also seeing some changes related to this in https://github.com/mozilla-mobile/fenix/pull/23168. @jonalmeida any particular reasoning we are using the StringRes over the string.

Reply:
So we can map the error to the ID for the string until we get to a piece of UI that can get the correct string (context.getString).

This is where it's used: https://searchfox.org/mozilla-mobile/rev/cf261343f08aea1a6a07a018e5942e30c54f7354/fenix/app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabsTrayLayout.kt#95